src/main/resources/templates/admin.html

```diff
@@ -5,415 +5,416 @@
+				<!--<div id="experiments" class="tab-pane fade">-->
+				<!--<p>&nbsp;</p>-->
+				<!--<table class="table table-responsive">-->
+				<!--<thead>-->
+				<!--<th th:text="#{admin.overview.header.experiment}">Experiment</th>-->
+				<!--<th th:text="#{admin.overview.header.team}">Team</th>-->
+				<!--<th th:text="#{admin.overview.header.status}">Status</th>-->
+				<!--<th th:text="#{admin.overview.header.nodesUtilized}">Nodes Utilized</th>-->
+				<!--<th th:text="#{admin.overview.header.hours}">Hours</th>-->
+				<!--<th>&lt;!&ndash; Actions &ndash;&gt;</th>-->
+				<!--</thead>-->
+				<!--<th:block th:each="expList : ${experimentMap}">-->
+				<!--<th:block th:each="expEntry : ${expList.value}">-->
+				<!--<tr th:if="${expEntry.status == 'READY'}">-->
+				<!--<td th:text="${expEntry.name}"></td>-->
+				<!--<td th:text="${teamManager.getTeamNameByTeamId(expEntry.teamId)}"></td>-->
+				<!--<td th:text="${expEntry.status}"></td>-->
+				<!--<td th:text="${expEntry.nodesCount}"></td>-->
+				<!--<td th:text="${expEntry.hoursRunning}"></td>-->
+				<!--<td>-->
+				<!--<a href="#" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Alert the experiment starter by sending email"><i class="fa fa-envelope-o"></i></a>-->
+				<!--<a href="#" th:href="@{/admin/experiments/remove/{expId}(expId=${expEntry.experimentId})}" class="btn btn-primary" data-toggle="tooltip" data-placement="bottom" title="Are you sure you want to permanently delete the experiment?"><i class="fa fa-trash"></i></a>-->
+				<!--</td>-->
+				<!--</tr>-->
+				<!--</th:block>-->
+				<!--</th:block>-->
+				<!--</table>-->
+				<!--</div>-->
+				<!--</div> &lt;!&ndash; tab content &ndash;&gt;-->
+				<!---->
+				<!--</div> &lt;!&ndash; panel body &ndash;&gt;-->
+				<!--</div> &lt;!&ndash; panel primary &ndash;&gt;-->
+				<!--</div> &lt;!&ndash; resourcesManagement &ndash;&gt;-->
+
+			</div> <!-- tab content -->
+		</div> <!-- row -->
+	<div class="cc-wraper">
+		<div class="cc-contactpop fixed">
+			<h1>Do you need Help? <span>Get in touch.</span></h1>
+			<div class="renseignement">
+				<i class="fa fa-times"></i>
+				<p>We will get back to you as soon as possible. </p>
 			</div>
-        </div> <!-- container -->
-		<div th:replace="fragments/footer:: footer"></div>
-		<object th:include="fragments/javascript :: javascript" th:remove="tag" />
-    </body>
+			<form>
+				<ul>
+					<li><i class=""></i> <span> Phone: </span>+65 6601 1056</li>
+					<li><i class=""></i><span>Email:</span><a href="mailto:support@ncl.sg"> support@ncl.sg</a></li>
+					<li><i class=""></i> <span> Address: </span><a href="https://www.google.com.sg/maps/dir//NUS+Computing,+13+Computing+Drive,+117417/@1.3437459,103.8240449,11z/data=!4m8!4m7!1m0!1m5!1m1!1s0x31da1af8c0faba33:0x9f0d02392be4dfbe!2m2!1d103.773846!2d1.295053?hl=en" >13 COMPUTING DRIVE, COM1 #01-16 Singapore 117417</a></li>
+				</ul>
+				<p>&nbsp;</p>
+			</form>
+		</div>
+		<!--</div>-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)